integrationlink/create.go

```diff
@@ -29,7 +30,7 @@ func Create(app string, integrationType scalingo.SCMType, integrationURL string,
 	}
 
 	repoLink, err := c.SCMRepoLinkShow(app)
-	if err != nil {
+	if scerr, ok := errors.ErrgoRoot(err).(*http.RequestFailedError); ok && scerr.Code != 404 {
```

Comment:
What happens if the error type is not `http.RequestFailedError`?

Reply:
Here, you are ignoring the error if it's a 404 or if it's anything else but a `http.RequestFailedError`. I think the intended behaviour is to ignore the error only in case of a 404, isn't it?

Reply:
I return errors only if it's not 404, if the error is a 404 the code is executed normally. 

I don't understand, it's written in the code ^^

Reply:
I mean that you MUST return if the error is not a `http.RequestFailedError`. And it's not the case.

Reply:
Hmmm ok